conf/uppmax.config

```diff
@@ -54,26 +60,18 @@ process {
   scratch = '$SNIC_TMP'
 }
 
-params {
-  save_reference = true
-
-  max_memory = 500.GB
-  max_cpus = 16
-  max_time = 240.h
-  // illumina iGenomes reference file paths on UPPMAX
-  igenomes_base = '/sw/data/igenomes/'
-}
-
-if (hostname ==~ "s[0-9][0-9]*") {
+if (hostname.startsWith("s")) {
   params.max_time = 700.h
   params.max_memory = 3880.GB
+  params.config_profile_description = 'UPPMAX (Snowy) cluster profile provided by nf-core/configs.'
 }
 
-if (hostname ==~ "i.*") {
+if (hostname.startsWith("i")) {
   params.max_memory = 250.GB
+  params.config_profile_description = 'UPPMAX (Irma) cluster profile provided by nf-core/configs.'
 }
 
-if (hostname ==~ "r.*") {
+if (hostname.startsWith("r")) {
```

Comment:
Why don't we use these as the defaults above if Rackham is the default machine?

Reply:
Yeah good question. I'll update it to do that instead.

Reply:
ah maybe because the default is Rackham / Bianca and not just Rackham.

Reply:
Then we should add a comment above, that those are for Bianca, while these are for Rackham, and split the descriptions too.